src/Dialect/ONNX/ONNXOps.td.inc

```diff
@@ -3812,6 +3812,7 @@ def ONNXPowOp:ONNX_Op<"Pow",
         return {20};
       }
     }];
+    let verifier = [{ return ::verify(*this); }];
```

Comment:
You should use the `hasVerifier` version instead. This usage of `verifier` will be obsolete soon.